lib/main.dart

```diff
@@ -30,40 +35,57 @@ class CogboardApp extends StatelessWidget {
         ChangeNotifierProvider.value(
           value: DashboardsProvider(),
         ),
-        ChangeNotifierProvider.value(
-          value: ConfigProvider(),
+        ChangeNotifierProxyProvider<SettingsProvider, ConfigProvider>(
+          create: (_) => ConfigProvider(),
+          update: (_, settingsProvider, configProvider) =>
+              configProvider.withSettingsPreferences(settingsProvider.settingsPreferences),
         ),
         ChangeNotifierProvider.value(
           value: FilterProvider(),
         ),
       ],
       child: MaterialApp(
-          title: 'Cogboard',
-          theme: ThemeData(
-            colorScheme: ColorScheme(
-                primary: Color(0xff81D4FA),
-                primaryVariant: Color(0xff81D4FA),
-                secondary: Color(0xffaed581),
-                secondaryVariant: Color(0xffaed581),
-                surface: Color(0xff202020),
-                background: Color(0xff121212),
-                error: Color(0xffcf6679),
-                onPrimary: Color(0xff000000),
-                onSecondary: Color(0xff000000),
-                onSurface: Color(0xffffffff),
-                onBackground: Color(0xffffffff),
-                onError: Color(0xff000000),
-                brightness: Brightness.dark),
-          ),
-          initialRoute: LoginScreen.routeName,
-          routes: {
-            LoginScreen.routeName: (ctx) => LoginScreen(),
-            DashboardsScreen.routeName: (ctx) => DashboardsScreen(),
-            SettingsScreen.routeName: (ctx) => SettingsScreen(),
-            WidgetDetailsScreen.routeName: (ctx) => WidgetDetailsScreen(),
-            DashboardItemScreen.routeName: (ctx) => DashboardItemScreen(),
-            AddConnectionScreen.routeName: (ctx) => AddConnectionScreen(),
-          }),
+        title: 'Cogboard',
+        theme: ThemeData(
+          colorScheme: ColorScheme(
+              primary: Color(0xff81D4FA),
+              primaryVariant: Color(0xff81D4FA),
+              secondary: Color(0xffaed581),
+              secondaryVariant: Color(0xffaed581),
+              surface: Color(0xff202020),
+              background: Color(0xff121212),
+              error: Color(0xffcf6679),
+              onPrimary: Color(0xff000000),
+              onSecondary: Color(0xff000000),
+              onSurface: Color(0xffffffff),
+              onBackground: Color(0xffffffff),
+              onError: Color(0xff000000),
+              brightness: Brightness.dark),
```

Comment:
Can we move ColorScheme to a constant? In some nice place - but not with the other constants.